ui/settings.py

```diff
@@ -1,20 +1,24 @@
+# This file houses all default settings for the Admin UI
+# to override please use a custom localsettings.py file
+
 # Flask configuration
 FLASK_LOGGING_ENABLED = True
 FLASK_PORT = 5000
+DEBUG = False
+
+STAT_REQ_FREQ = 910
+
+# REST host information
+REST_HOST = 'http://rest:5343'
```

Comment:
Change this to `localhost`, I think the `docker/` settings should still be `rest`, but the travis unit tests that run vanilla centos or ubuntu look for `localhost` as the service endpoint (which is causing the Travis build to partially fail)